apps/api/src/sockets/socket.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@nestjs/common'
+import { SocketGateway } from './gateway'
+
+@Injectable()
+export class WebSocketService {
```

Comment:
[naming guideline] NotificationService would be a better name to show the purpose